grplot/features/optimizer/optimizer_key.py

```diff
@@ -3,16 +3,17 @@
 
 def optimizer_key(var_list):
     key = numpy.array([])
-    if type(var_list) in [list, numpy.ndarray]:
-        for var in var_list:
-            if type(var) == str:
-                key = numpy.concatenate([key, numpy.array([var])])
-            elif type(var) == list:
-                key = numpy.concatenate([key, numpy.array(var)])
-            elif type(var) == numpy.ndarray:
-                key = numpy.concatenate([key, var])
-            else:
-                pass
-    else:
-        pass
+    if type(var_list) not in [list, numpy.ndarray]:
+        return key
+
+    for var in var_list:
+        if type(var) == str:
```

Comment:
Yang ini?

Reply:
eh aman deh, itu buat var in var_list wkwkwk. rapihin dikit aja yon, samain cara enter tiap linenya kayak yang gua bikin biar clean aja. gas merge abis itu.

Reply:
ehh bentar maksud lu gk ada enternya? klo misal gk ada enter gitu ngurangin readability codenya, lu coba cek seaborn  dia biasanya antar if itu ada enternya buat ningkatin readability paling yang antar if elif sma else yang bisa di gk adain enter

Reply:
biar sama aja format codenya yon, lagian di python juga basisnya indentasi jadi udah keliatan juga sih.